### PR TITLE
Adding ncflag

### PR DESCRIPTION
**Pull request by 5tefan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by 5tefan** [comment]
@conda-forge/help-python, I think this is ready for review. Thank you!

**Comment by 5tefan** [comment]
@conda-forge/help-python I think this is ready for review. Thank you!

File: recipes/ncflag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - python -m unittest discover -s .
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pypandoc
+    - numpy
+    - netcdf4
+    - click >=7.0,<8.0
```
[the remark this answers is not in the document]
Comment (5tefan) [reply recipes/ncflag/meta.yaml:27]: Agree, only python, pip, and pypandoc would be required, but the reason for the others is that the `python -m unittest discover -s .` is done in the build. This approach was recommended for a similar recipe here: https://github.com/conda-forge/staged-recipes/pull/14464#discussion_r607614888, is there a better way to do this?

[the remark this answers is not in the document]
Comment (isuruf) [reply recipes/ncflag/meta.yaml:27]: That's not the correct way. See conda-build docs on `test: source_files`. cc @xhochy

**Comment by 5tefan** [comment]
Thanks, looks like source_files was exactly what was missing!

**Pull request merged**